package.json

```diff
@@ -39,7 +39,10 @@
 		"@steeze-ui/heroicons": "^2.2.3",
 		"@steeze-ui/icons": "^2.1.0",
 		"@steeze-ui/svelte-icon": "^1.5.0",
-		"langchain": "^0.0.198",
-		"tiktoken": "^1.0.11"
+		"build": "^0.1.4",
```

Comment:
I don't think we using some of these dependencies. Can you clean some of this up? Ex: build and zod